charts/kkr/wallet-api/templates/ingress/ingress.api.private.yaml

```
@@ -0,0 +1,67 @@
+{{- if .Values.ingress.enabled -}}
+{{- $fullName := include "wallet-api.fullname" . -}}
+{{- $svcPort := .Values.service.port -}}
+{{- if and .Values.ingress.className (not (semverCompare ">=1.18-0" .Capabilities.KubeVersion.GitVersion)) }}
+  {{- if not (hasKey .Values.ingress.annotations "kubernetes.io/ingress.class") }}
+  {{- $_ := set .Values.ingress.annotations "kubernetes.io/ingress.class" .Values.ingress.className}}
+  {{- end }}
+{{- end }}
+{{- if semverCompare ">=1.19-0" .Capabilities.KubeVersion.GitVersion -}}
+apiVersion: networking.k8s.io/v1
+{{- else if semverCompare ">=1.14-0" .Capabilities.KubeVersion.GitVersion -}}
+apiVersion: networking.k8s.io/v1beta1
+{{- else -}}
+apiVersion: extensions/v1beta1
+{{- end }}
+kind: Ingress
+metadata:
+  name: {{ $fullName }}
+  labels:
+    {{- include "wallet-api.labels" . | nindent 4 }}
+  annotations:
+    konghq.com/plugins: {{ $fullName }}-cors,{{ $fullName }}-validate-auth-server-token
```

```suggestion
    konghq.com/plugins: {{ include "wallet-api.corsPluginName" . }},{{ include "wallet-api.validateAuthServerTokenPluginName" . }}
```
`_helper` 에 적용한 내용을 반영하려면, 이 부분에 함께 적용되야 맞을 것 같습니다. (그리고 이에 맞게 `_helper.tpl` 이 업데이트되어야 합니다. 이 부분은 추가로 가이드 해드리겠습니다.)

---

네 해당 부분 `_helper.tpl`로 분리하였다가  argocd에서 sync error가 발생하여서 일단 revert 해두었습니다.
tx 관련 이슈가 발생했는데 연관되어 있는 부분인가 싶어서  revert 했는데, 아닌 것으로 파악되어서 다시 적용한 상태 올려두도록 하겠습니댜.